tensorflow/lite/micro/micro_allocator.h

```diff
@@ -240,6 +247,12 @@ class MicroAllocator {
 
   ErrorReporter* error_reporter() const;
 
+  // Returns the first subgraph from the model.
+  const SubGraph* GetSubGraphFromModel(const Model* model);
```

Comment:
Can you explain the need for this GetSubGraphFromModel function?

Reply:
As part of this PrepareNodeAndRegistrationDataFromFlatbuffer() function , we called AllocateIntermediateTfLiteTensorFromFlatbuffer() function to get intermediate tensor and intern this function uses GetSubGraphFromModel to init TfLiteTensorFromFlatbuffer